ansible/roles/stack-sunbird/templates/transformer_industry_feedback.xml

```diff
@@ -0,0 +1,3 @@
of_contact" required="true()" type="string"/><bind constraint="regex(., '^[0-9]{10}$')" jr:constraintMsg="Please enter valid mobile number." nodeset="/data/personal_info/mobile_number" required="true()" type="string"/><bind nodeset="/data/personal_info/industry_location" required="true()" type="string"/><bind nodeset="/data/personal_info/industry_type" required="true()" type="string"/><bind nodeset="/data/personal_info/industry_sector_services" relevant=" /data/personal_info/industry_type  = 'Services'" required="true()" type="string"/><bind nodeset="/data/personal_info/industry_sector_manufacturing" relevant=" /data/personal_info/industry_type  = 'Manufacturing'" type="string"/><bind nodeset="/data/personal_info/job_role" required="true()" type="string"/><bind nodeset="/data/personal_info/min_qual" required="true()" type="string"/><bind jr:constraintMsg="Required" nodeset="/data/industry_strength/resume_ops" required="true()" type="string"/><bind constraint="regex(., '^[0-9]{1,}$')" jr:constraintMsg="Required" nodeset="/data/industry_strength/no_emp_per" required="true()" type="string"/><bind constraint="regex(., '^[0-9]{1,}$')" jr:constraintMsg="Please enter a valid number" nodeset="/data/industry_strength/no_emp_con" required="true()" type="string"/><bind constraint="regex(., '^[0-9]{1,}$')" jr:constraintMsg="Please enter a valid number" nodeset="/data/industry_strength/emp_migrant" type="string"/><bind constraint="regex(., '^[0-9]{1,}$')" jr:constraintMsg="Please enter a valid number" nodeset="/data/industry_strength/emp_current_per" required="true()" type="string"/><bind constraint="regex(., '^[0-9]{1,}$')" jr:constraintMsg="Please enter a valid number" nodeset="/data/industry_strength/emp_current_con" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/average_hiring" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/current_requirement_number" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/time_frame" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/hiring_local" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/hiring_apprentices" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/apprentices_number" relevant=" /data/hiring_requirements/hiring_apprentices = 'Yes'" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/usage_rozgar" required="true()" type="string"/><bind nodeset="/data/hiring_requirements/reason_rozgar_usage" relevant=" /data/hiring_requirements/usage_rozgar = 'No'" required="true()" type="string"/><bind jr:preload="uid" nodeset="/data/meta/instanceID" readonly="true()" type="string"/></model></h:head><h:body><group ref="/data/personal_details"><label>Introduction</label><input ref="/data/personal_details/form_intro"><label>- This survey is being conducted by the Saksham Haryana team, CM Office in light of the current labour crisis being faced by establishments.
```

Comment:
This file is impossible to read. Can we format it to make it more readable? https://jsonformatter.org/xml-formatter

Reply:
Converted the file content into proper XML format.